exercises/accumulate/AccumulateTests.elm

```diff
@@ -12,18 +15,21 @@ square x =
 
 tests : Test
 tests =
-    suite "Accumulate"
+    describe "Accumulate"
         [ test "[]] Accumulate"
-            (assertEqual [] (accumulate square []))
+            (\() -> Expect.equal [] (accumulate square []))
```

Comment:
Ok, you got me. I figured how to get 95% of it in bulk, and elm-format helped me sort out the  rest pretty quickly.